content/integrations/awspolly.md

```diff
@@ -4,6 +4,7 @@ integration_title: AWS Polly
 kind: integration
 git_integration_title: amazon_polly
 newhlevel: true
+description: "Track key AWS Polly metrics."
```

Comment:
Can we expand on this? What kind of metrics?

Reply:
🆗